TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -187,11 +189,27 @@ public class TwilioVerifyBuilder {
     return self
   }
 
+  /// Set the accessGroup that will be used for the Keychain storage.
+  /// - Parameter accessGroup: Used to specify the access group for the [kSecAttrAccessGroup](https://developer.apple.com/documentation/security/ksecattraccessgroup?language=swift) of the KeyChain
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/3d084cdf558fe2c3c223d0f0881d860cd43299b1/TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift#L193"></span>Line should be 195 characters or less: currently 196 characters<br />
<code>line_length</code> <code>TwilioVerify.swift:193</code></td>
    </tr>
  </tbody>
</table>